packages/components/src/styles/table-cell/table-cell.css

```diff
@@ -0,0 +1,37 @@
+/* 
+* Remove inherited styles from table before applying our styles.
+*/
+.sd-table {
+  all: unset;
+  display: table;
```

Comment:
Why did you not add the tailwind classes?

@apply table border-collapse border-spacing-0



<br class="Apple-interchange-newline">

Reply:
Good question...Will change this.

Reply:
Moved to new sd-table component and updated to tw classes.